### PR TITLE
Task30

### PR DESCRIPTION
**Pull request by olgamorozova1**

### CODE REVIEW

File: src/main/java/test/LoginTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.concurrent.TimeUnit;
+
+public class LoginTest {
+    private WebDriver driver;
+    public static final String URL = "https://www.tut.by/";
```
Comment (KirillKarpov) [inline src/main/java/test/LoginTest.java:15]: Do we need public there?

File: src/main/java/page/MainPage.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package page;
```
Comment (KirillKarpov) [inline src/main/java/page/MainPage.java:1]: Please create package under folder **java** and move all your packages(test and page) there. The package name could be  org.issoft.automation

File: pom.xml [hunk 3]
```diff
@@ -7,6 +7,41 @@
...
+            <artifactId>junit-jupiter-params</artifactId>
+            <version>5.7.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.7.0</version>
+            <scope>test</scope>
+        </dependency>
```
[the remark this answers is not in the document]
Comment (olgamorozova1) [reply pom.xml]: Removed

File: src/main/java/test/LoginTest.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
+        driver.get(URL);
+    }
+
+    @Test
+    public void loginWithCorrectCredentials() {
+        MainPage mainPage = new MainPage(driver);
+        mainPage.clickLoginLink();
+        mainPage.login("seleniumtests@tut.by", "123456789zxcvbn");
```
[the remark this answers is not in the document]
Comment (olgamorozova1) [reply src/main/java/test/LoginTest.java]: Created constants

File: src/main/java/test/LoginTest.java (hunk 1 above)
Comment (olgamorozova1) [reply src/main/java/test/LoginTest.java]: No, made private all constants.

File: src/main/java/page/MainPage.java (hunk 2 above)
Comment (olgamorozova1) [reply src/main/java/page/MainPage.java]: Created new package

**Review by KirillKarpov — APPROVED** [review]

**Pull request merged**